pkgs/applications/altcoins/ledger-live-desktop/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+ stdenv,
+ appimage-run,
+ fetchurl,
+ runtimeShell,
+}:
+
+stdenv.mkDerivation rec {
+  github-user = "LedgerHQ";
+  pname = "ledger-live-desktop";
+  version = "1.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/${github-user}/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
+    sha256 = "0b919ilvv3zi17fnzngnnmg28dxqlq0dvj3fdvb50kh3ibrhdcfz";
+  };
+
+  buildInputs = [ appimage-run];
+
+  unpackPhase = ":";
+
+  installPhase = ''
+    mkdir -p $out/{bin,share}
+    cp $src $out/share/${pname}
+    echo "#!${runtimeShell}" > $out/bin/${pname}
+    echo "${appimage-run}/bin/appimage-run $out/share/${pname}" >> $out/bin/${pname}
+    chmod +x $out/bin/${pname} $out/share/${pname}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Ledger Live (Desktop)";
+    longDescription = ''
+      Ledger Live (Desktop)
+    '';
+    homepage = https://www.ledger.com/live;
```

Comment:
quotes round urls

Reply:
@alexarice why i thought urls were first class?

Reply:
They are first class. There is an ongoing discussion on whether to try to avoid their usage though I looked at it yesterday and it is quite ongoing so you could leave this if you want